Makefile

```diff
@@ -1,11 +1,8 @@
 build:
-	docker build -t stackstorm/stackstorm:latest st2-upstart
+	docker build -t stackstorm/stackstorm:latest images/eval
```

Comment:
I thought we agreed on `stackstorm` naming for all-in-one image.
And it's bigger than just `st2`, since it contains st2web and more.